pkg/render/node_test.go

```diff
@@ -199,6 +199,8 @@ var _ = Describe("Node rendering tests", func() {
   "plugins": [
```

Comment:
I think we need some tests that check non-zero values

Reply:
Added some values to validation_test.go now.
Also checking rendering of custom values in node_test.go